src/ol_infrastructure/applications/ocw_site/404.html

```diff
@@ -1,352 +1,148 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="{{ $.Site.Language.Lang }}">
```

Comment:
Is this a Hugo template tag? It doesn't seem like this will be rendered properly when loaded via Fastly

Reply:
Sorry this escaped my go through. Fixed